CHANGELOG

```diff
@@ -1,3 +1,28 @@
+Version 0.5.0 (2024-11-01)
+--------------------------
+Disable asynchronous deletes for Hudi
+Upgrade dependencies
+DynamoDB runtime dependencies for Delta S3 multi-writer
+PDP-1323 Handle initialization errors for GCP
+PDP-1324 - Handle initialization errors for lake loader Delta Azure
+Upgrade common-streams to 0.8.0-M5
+Delete files asynchronously
+Upgrade common-streams 0.8.0-M4
+Avoid error on duplicate view name
+Add option to exit on missing Iglu schemas
+common-streams 0.8.x with refactored health monitoring
+Create table concurrently with subscribing to stream of events
+Iceberg fail fast if missing permissions on the catalog
+Make alert messages more human-readable
+Hudi loader should fail early if missing permissions on Glue catalog
+Add alert & retry for delta/s3 initialization
+Implement alerting and retrying mechanisms
+Bump aws-hudi to 1.0.0-beta2
+Bump hudi to 0.15.0
+Allow disregarding Iglu field's nullability when creating output columns
+Extend health probe to report unhealthy on more error scenarios
+Fix bad rows resizing
```

Comment:
For all earlier changelogs we have references to the relevant issue/PR, e.g.

```
Disable asynchronous deletes for Hudi (#94)
```

I think we should keep the references.